axon-5/api-changes.md

```diff
@@ -727,17 +727,39 @@ This section contains two tables:
+| org.axonframework.eventsourcing.conflictresolution.ConflictResolver                      | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.eventsourcing.conflictresolution.Conflicts                             | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.eventsourcing.conflictresolution.ContextAwareConflictExceptionSupplier | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.eventsourcing.conflictresolution.DefaultConflictDescription            | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.eventsourcing.conflictresolution.DefaultConflictResolver               | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.eventsourcing.conflictresolution.NoConflictResolver                    | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.modelling.command.ConflictingAggregateVersionException                 | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.modelling.command.ConflictingModificationException                     | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.modelling.command.TargetAggregateVersion                               | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+| org.axonframework.modelling.command.VersionedAggregateIdentifier                         | No longer supported in Axon Framework 5 due to limited use by the community.                                                                   |
+
+### Marked for removal Classes
+
+All classes in this table have been moved to the legacy package for ease in migration.
+However, they will eventually be removed entirely from Axon Framework 5, as we expect users to migrate to the new (and per class described) approoach.
+
+| Class                                                                    |
+|--------------------------------------------------------------------------|
+| org.axonframework.modelling.command.Repository                           |
```

Comment:
If is it for removal, why you didn;t mark it as `@Deprecated(since = "5.0.0", forRemoval = true)`?

Reply:
I wasn't aware `forRemoval` existed. It was apparently introduced in Java 9. Nice!